src/library_manager/lib_manager.c

```diff
@@ -366,11 +379,10 @@ int lib_manager_register_module(struct sof_man_fw_desc *desc, int module_id)
 	return ret;
 }
 
-static void lib_manager_dma_buffer_update(struct lib_manager_dma_buf *buffer, uintptr_t addr,
+static void lib_manager_dma_buffer_update(struct lib_manager_dma_buf *buffer,
 					  uint32_t size)
 {
 	buffer->size = size;
-	buffer->addr = addr;
 	buffer->w_ptr = buffer->addr;
```

Comment:
@lyakh  Is this removal intentional? This seems like a bigger change than just adding annotations?

Reply:
yes, intentional. Yeah, there are some fixes like this. This assignment is redundant, this function was always called with `(buf, buf->addr,...)` so it was an assignment to itself. That's why I removed that `addr` argument completely. The change seemed small enough to me to integrate it...

Reply:
Ack, @jxstelter @pjdobrowolski @softwarecki can you check?

Reply:
Looks good.